.circleci/config.yml

```diff
@@ -166,15 +176,22 @@ jobs:
       - slack/status:
           mentions: "nick"
 
+parameters:
+  verify_release:
+    default: false
+    type: boolean
+
 workflows:
   version: 2
   shellcheck:
+    unless: << pipeline.parameters.verify_release >>
```

Comment:
Is this valid yaml to have those angle brackets unescaped?

Reply:
Yep! (I think `>` will cause problems at string start, but `<` is fine)

https://support.circleci.com/hc/en-us/articles/360043638052-Conditional-steps-in-jobs-and-conditional-workflows